Resources/Locale/en-US/SimpleStation14/magic/hippocraticoath.ftl

```diff
@@ -0,0 +1,35 @@
+old-asclepius-binding-hippocratic-oath-progress-3 = If I do not violate this oath, may I enjoy life and art, respected while I live and remembered with affection thereafter. May I always act so as to preserve the finest traditions of my calling and may I long experience the joy of healing those who seek my help.
+
+# Oath from Wikipedia (https://en.wikipedia.org/wiki/Hippocratic_Oath#Modern_versions_and_relevance)
+# Average time in ms for me to read each character in each verse:
+# 23.87
+asclepius-binding-hippocratic-oath-progress-0 = I swear to fulfill, to the best of my ability and judgment, this covenant:
+# 21.05
+asclepius-binding-hippocratic-oath-progress-1 = I will respect the hard-won scientific gains of those physicians in whose steps I walk, and gladly share such knowledge as is mine with those who are to follow.
+# 23.404
+asclepius-binding-hippocratic-oath-progress-2 = I will apply, for the benefit of the sick, all measures that are required, avoiding those twin traps of overtreatment and therapeutic nihilism.
+# 20.986
+asclepius-binding-hippocratic-oath-progress-3 = I will remember that there is art to medicine as well as science, and that warmth, sympathy, and understanding may outweigh the surgeon's knife or the chemist's drug.
+# 18.05
+asclepius-binding-hippocratic-oath-progress-4 = I will not be ashamed to say "I know not", nor will I fail to call in my colleagues when the skills of another are needed for a patient's recovery.
+# 17.677
+asclepius-binding-hippocratic-oath-progress-5 = I will respect the privacy of my patients, for their problems are not disclosed to me that the world may know. Most especially must I tread with care in matters of life and death. If it is given me to save a life, all thanks. But it may also be within my power to take a life; this awesome responsibility must be faced with great humbleness and awareness of my own frailty. Above all, I must not play at God.
+# 19.844
+asclepius-binding-hippocratic-oath-progress-6 = I will remember that I do not treat a fever chart, a cancerous growth, but a sick human being, whose illness may affect the person's family and economic stability. My responsibility includes these related problems, if I am to care adequately for the sick.
+# 22.157
+asclepius-binding-hippocratic-oath-progress-7 = I will prevent disease whenever I can, for prevention is preferable to cure.
+# 21.322
+asclepius-binding-hippocratic-oath-progress-8 = I will remember that I remain a member of society, with special obligations to all my fellow human beings, those sound of mind and body as well as the infirm.
+# 19.55
+asclepius-binding-hippocratic-oath-progress-9 = If I do not violate this oath, may I enjoy life and art, respected while I live and remembered with affection thereafter. May I always act so as to preserve the finest traditions of my calling and may I long experience the joy of healing those who seek my help.
+# Average time for the whole oath:
+## 20.791
```

Comment:
I don't quite understand the numbers here- isn't the time being calculated by the length of each string? Is 20.791 the time-per-character, but for each independent statement?

Reply:
I timed how long it took me to read each loc and averaged the times to get 20.791ms per character
Another reference too